exercises/practice/darts/src/darts.cob

```diff
@@ -0,0 +1,10 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. LEAP.

```

Comment:
Actually there's an issue there, it says LEAP instead of darts

Reply:
@0xE282B0 could you push another PR to fix this?

Reply:
Sure, sorry.